plugins/heatserts/README.md

```diff
@@ -0,0 +1,51 @@
+# Heatserts
+
+Generates a hole that accepts a heatsert. Heatserts are often used in 3D printed parts. Values included that fit my M3, M4, M5 and M6 heatserts, YMMV.
+
+## Installation
+
+```
+pip install -e "git+https://github.com/CadQuery/cadquery-plugins.git#egg=heatsert&subdirectory=plugins/heatsert"
+```
+You can also clone the repository of the plugin and run in the repository the following command:
+```
+python setup.py install
+```
+
+## Dependencies
+
+This plugin has no dependencies other than the cadquery library.
+
+## Usage
+
+To use this plugin after it has been installed, import it to automatically patch the `heatsert` method into the `cadquery.Workplane` class. The `heatsert` function should be available after import, but be sure to import `cadquery` first.
+
+Also see the docstring on the `heatsert` method for a description of it's arguments.
+
+```python
+import cadquery as cq
+import heatserts  # Adds the heatsert function to cadquery.Workplane
+
+result = (
+    cq.Workplane()
+    .box(100, 100, 10)
+    .faces(">Z")
+    .wires()
+    .toPending()
+    .offset2D(-5, forConstruction=True)
+    .vertices()
+    .heatsert("M4")
+)
+```
+
+The user can fine tune the dimensions of the heatsert hole (to suit their brand of heatserts or to add clearance at the bottom of the hole), or add new heatserts. To add a new M8 heatsert (numbers are not accurate):
+```python
+import cadquery as cq
+import heatserts.heatserts as heatsert_module
```

Comment:
This line fails when using the plugin after installing it. I think that import will only work in the tests.

Reply:
This is resolved when disabling the auto reload in CQ-editor, right?

I haven't tested in CQ-editor myself because it's a bit of a pain to do in my dev environment.

Reply:
I don't think this one is related to CQ-editor. It's the `heatserts.heatserts` part of the import that's the problem. You can test it from the Python REPL after installing the plugin with pip.

Reply:
Ah, now I see, thanks for catching that.